PhoenixPointModLoader/LegacyPhoenixPointMod.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+
+namespace PhoenixPointModLoader
+{
+    class LegacyPhoenixPointMod : IPhoenixPointMod
```

Comment:
I like the idea of wrapping legacy mods with this class, clever! I understand it's nitpicking; please try to be consistent with  code conventions in the rest of the project. 

* Private fields should start with an underscore.
* Method names should be pascal case.